dovecot/entrypoint.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/ash
```

Comment:
Why not bash? :thinking:

Reply:
Because B stands for Bloat :joy_cat: 

Jokes apart, Ash is the default shell in Alpine.

Reply:
:+1: